design-system/materials/internals/definition.yaml

```diff
@@ -397,6 +405,18 @@ decisionGroupsByTheme:
           choice: 'none'
         border-for-collapsible-panel-header-icon-when-disabled:
           choice: '1px solid var(--color-neutral)'
+        border-for-stamp-when-error:
+          choice: color-error
+        border-for-stamp-when-warning:
+          choice: color-warning
+        border-for-stamp-as-positive:
+          choice: color-primary-40
+        border-for-stamp-as-information:
+          choice: color-info
+        border-for-stamp-as-primary:
+          choice: color-primary-25
+        border-for-stamp-as-secondary:
+          choice: color-neutral-60
```

Comment:
This should be renames to `border-color-for-*' and moved to the block below (`borderColors`) and also for the new theme.